### PR TITLE
exit with error msg if log4j.properties file not exist.

### PR DESCRIPTION
**Pull request by sunghyuk**
web-server and executor-server redirected out and err stream to logstream therefore log4j.properties is mandatory.
it differs from solo-server.

```
public static void main(final String[] args) throws Exception {
  // Redirect all std out and err messages into log4j
  StdOutErrRedirect.redirectOutAndErrToLog();
```

so if log4j.properties file not exist, startup script should exit with file missing error and it more helpful than stackoverflow error. 

#860
#914

### CODE REVIEW

File: azkaban-exec-server/src/main/bash/azkaban-executor-start.sh [hunk 1]
```diff
@@ -50,9 +50,12 @@ if [[ -z "$AZKABAN_OPTS" ]]; then
 fi
 # Set the log4j configuration file
 if [ -f $conf/log4j.properties ]; then
-  AZKABAN_OPTS="$AZKABAN_OPTS -Dlog4j.configuration=file:$conf/log4j.properties"
+  AZKABAN_OPTS="$AZKABAN_OPTS -Dlog4j.configuration=file:$conf/log4j.properties -Dlog4j.log.dir=$azkaban_dir/logs"
+else
+  echo "Exit with error: $conf/log4j.properties file must be exist."
```
Comment (HappyRay) [inline azkaban-exec-server/src/main/bash/azkaban-executor-start.sh]: " file doesn't exist"?

**Comment by HappyRay** [comment]
LGTM otherwise.

File: azkaban-exec-server/src/main/bash/azkaban-executor-start.sh (hunk 1 above)
Comment (sunghyuk) [reply azkaban-exec-server/src/main/bash/azkaban-executor-start.sh]: sorry, I'm not native. fix please

Comment (HappyRay) [reply azkaban-exec-server/src/main/bash/azkaban-executor-start.sh]: No problem.  Could you update your PR with the suggested change?

**Pull request merged**